src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_cassandra.json

```diff
@@ -0,0 +1,18 @@
+[
+ {
+  "op": "add",
+  "path": "/ValueTypes/AWS::Cassandra::Keyspace.KeyspaceName",
+  "value": {
+   "StringMax": 255,
```

Comment:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cassandra-keyspace.html#cfn-cassandra-keyspace-keyspacename


We are pulling this regex which doesn't match the described length.